packages/api/src/controllers/customAnimalTypeController.js

```diff
@@ -23,7 +23,7 @@ const customAnimalTypeController = {
     return async (req, res) => {
```

Comment:
These two changes to custom animal type are unrelated; I just noticed these points in discussions of other animal-related PRs and realized they were missing here.